### PR TITLE
Using a new API

### PR DESCRIPTION
**Pull request by juniorRubyist**

### CODE REVIEW

File: lib/tts.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@
 require 'uri'
 
 module Tts
-  @@default_url = "http://translate.google.com/translate_tts"
+  @@default_url = "http://soundoftext.com/static/sounds/en/"
```
Comment (qichunren) [inline lib/tts.rb]: @@default_url = "http://soundoftext.com/static/sounds/en/" should be

@@default_url = "http://soundoftext.com/static/sounds"

File: lib/tts.rb [hunk 2]
```diff
@@ -58,7 +58,7 @@ def to_valid_fn
   def to_url lang
     langs = ['af', 'ar', 'az', 'be', 'bg', 'bn', 'ca', 'cs', 'cy', 'da', 'de', 'el', 'en', 'en_us', 'en_gb', 'en_au', 'eo', 'es', 'et', 'eu', 'fa', 'fi', 'fr', 'ga', 'gl', 'gu', 'hi', 'hr', 'ht', 'hu', 'id', 'is', 'it', 'iw', 'ja', 'ka', 'kn', 'ko', 'la', 'lt', 'lv', 'mk', 'ms', 'mt', 'nl', 'no', 'pl', 'pt', 'ro', 'ru', 'sk', 'sl', 'sq', 'sr', 'sv', 'sw', 'ta', 'te', 'th', 'tl', 'tr', 'uk', 'ur', 'vi', 'yi', 'zh', 'zh-CN', 'zh-TW']
     raise "Not accepted language, accpeted are #{langs * ","}" unless langs.include? lang
-    base = "#{Tts.server_url}?tl=#{lang}&ie=UTF-8&client=t&q=#{URI.escape self}"
+    base = "#{Tts.server_url}/#{lang}/#{URI.escape self}"
```
Comment (qichunren) [inline lib/tts.rb]: base = "#{Tts.server_url}/#{lang}/#{URI.escape self}" should be

base = "#{Tts.server_url}/#{lang}/#{URI.escape self}.mp3"

**Comment by qichunren** [comment]
And this API is only usable after you reqeust text to mp3 from its site http://soundoftext.com/, because it cached request result. Otherwhile API url response 404.

**Comment by c2h2** [comment]
google service is working normally for now... closing...

**Pull request closed without merging**